From e21ffabe83e98de8d46cde0196bc6d6b2b12bc28 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Sat, 4 Apr 2020 18:27:42 +0800
Subject: [PATCH] ARM: configs: Add rv1126-evb-ddr3-v10-tb.config

Demo thunder boot config.
Create by:
./scripts/diffconfig -m | sort > arch/arm/configs/rv1126-evb-ddr3-v10-tb.config

Usage:
make ARCH=arm rv1109_defconfig rv1126-evb-ddr3-v10-tb.config

Change-Id: I69f2131ec3235bd50a97641d12a003d2c074bb21
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 .../arm/configs/rv1126-evb-ddr3-v10-tb.config | 67 +++++++++++++++++++
 1 file changed, 67 insertions(+)
 create mode 100644 arch/arm/configs/rv1126-evb-ddr3-v10-tb.config

diff --git a/arch/arm/configs/rv1126-evb-ddr3-v10-tb.config b/arch/arm/configs/rv1126-evb-ddr3-v10-tb.config
new file mode 100644
index 000000000000..f9434a9b2959
--- /dev/null
+++ b/arch/arm/configs/rv1126-evb-ddr3-v10-tb.config
@@ -0,0 +1,67 @@
+CONFIG_BT_HCIUART=m
+CONFIG_BT=m
+CONFIG_CFG80211=m
+# CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND is not set
+CONFIG_CPU_FREQ_DEFAULT_GOV_PERFORMANCE=y
+# CONFIG_DEBUG_FS is not set
+CONFIG_DRM_PANEL_SIMPLE_OF_ONLY=y
+CONFIG_DTC_OMIT_DISABLED=y
+CONFIG_DTC_OMIT_EMPTY=y
+# CONFIG_DTC_SYMBOLS is not set
+CONFIG_DWMAC_ROCKCHIP=m
+CONFIG_INITCALL_ASYNC=y
+# CONFIG_INPUT_TOUCHSCREEN is not set
+CONFIG_MAC80211=m
+CONFIG_MII=m
+CONFIG_MMC_DW_ROCKCHIP=m
+# CONFIG_PERF_EVENTS is not set
+CONFIG_PHY_ROCKCHIP_NANENG_USB2=m
+CONFIG_PRINTK_TIME_FROM_ARM_ARCH_TIMER=y
+CONFIG_RFKILL=m
+CONFIG_RFKILL_RK=m
+CONFIG_ROCKCHIP_HW_DECOMPRESS=y
+CONFIG_ROCKCHIP_RAMDISK=y
+CONFIG_ROCKCHIP_THUNDER_BOOT=y
+CONFIG_ROMFS_BACKED_BY_BLOCK=y
+# CONFIG_ROMFS_BACKED_BY_BOTH is not set
+# CONFIG_ROMFS_BACKED_BY_MTD is not set
+CONFIG_ROMFS_FS=y
+CONFIG_ROMFS_ON_BLOCK=y
+CONFIG_SCSI=m
+CONFIG_SCSI_MOD=m
+# CONFIG_SLUB_SYSFS is not set
+CONFIG_SND_ALOOP=m
+CONFIG_SND_DMAENGINE_PCM=m
+CONFIG_SND_HRTIMER=m
+CONFIG_SND=m
+CONFIG_SND_PCM=m
+CONFIG_SND_SIMPLE_CARD=m
+CONFIG_SND_SIMPLE_CARD_UTILS=m
+CONFIG_SND_SOC_DUMMY_CODEC=m
+CONFIG_SND_SOC_I2C_AND_SPI=m
+CONFIG_SND_SOC=m
+CONFIG_SND_SOC_RK817=m
+CONFIG_SND_SOC_ROCKCHIP_I2S=m
+CONFIG_SND_SOC_ROCKCHIP_I2S_TDM=m
+CONFIG_SND_SOC_ROCKCHIP=m
+CONFIG_SND_SOC_ROCKCHIP_PDM=m
+CONFIG_SND_TIMER=m
+CONFIG_SOUND=m
+CONFIG_STMMAC_ETH=m
+CONFIG_STMMAC_PLATFORM=m
+# CONFIG_TEE is not set
+CONFIG_USB_DWC3=m
+CONFIG_USB_DWC3_OF_SIMPLE=m
+CONFIG_USB_DWC3_ROCKCHIP_INNO=m
+CONFIG_USB_EHCI_HCD=m
+CONFIG_USB_EHCI_HCD_PLATFORM=m
+CONFIG_USB_HID=m
+# CONFIG_USB_KBD is not set
+CONFIG_USB=m
+# CONFIG_USB_MOUSE is not set
+# CONFIG_USB_NET_DRIVERS is not set
+CONFIG_USB_OHCI_HCD=m
+CONFIG_USB_OHCI_HCD_PLATFORM=m
+CONFIG_USB_XHCI_HCD=m
+CONFIG_USB_XHCI_PLATFORM=m
+CONFIG_WL_ROCKCHIP=m
-- 
2.35.3

